cli/package.json

```diff
@@ -22,6 +22,14 @@
   "bin": {
     "create-t3-app": "./dist/index.js"
   },
+  "files": [
+    "dist",
+    "template",
+    "README.md",
+    "CHANGELOG.md",
+    ".yarnrc.yml",
+    "package.json"
+  ],
```

Comment:
not related to app-dir, just minimizing the bundle size and we don't need to include src